src/System.Private.CoreLib/shared/Internal/Runtime/Augments/RuntimeThread.cs

```diff
@@ -0,0 +1,56 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Runtime.ConstrainedExecution;
+using System.Runtime.InteropServices;
+using System.Security;
+using System.Threading;
+
+namespace Internal.Runtime.Augments
+{
+    public class RuntimeThread : CriticalFinalizerObject
```

Comment:
Presumably this will go away once corefx is updated to depend on just Thread and then coreclr ingests that updated version of corefx for its testing?

Reply:
Yep, that's the plan. I tested CoreCLR/CoreRT compilation without this file and it passed.